runner/runners/output.go

```diff
@@ -32,3 +40,61 @@ func (o *nullOutput) Write(p []byte) (int, error) {
 func (o *nullOutput) Close() error {
 	return nil
 }
+
+type localOutputCreator struct {
+	hostname string
+}
+
+// Create a new OutputCreator
+func NewOutputCreator(tmp *temp.TempDir) (runner.OutputCreator, error) {
```

Comment:
Should this be called NewLocalOutputCreator?  It creates a specific implementation of the interface.  Suggest changing the name and update comments